watertap/unit_models/uv_aop.py

```diff
@@ -431,11 +436,82 @@ def build(self):
         def eq_uv_dose(b):
             return b.uv_dose == b.uv_intensity * b.exposure_time
 
-        @self.Constraint(
-            doc="Constraint for UV absorbance",
-        )
-        def eq_UVA(b):
-            return b.UVA == -log10(b.UVT) / (1 * pyunits.cm)
+        # ---------------------------------------------------------------------
+        if self.config.uv_dose_type == UVDoseType.calculated:
+            self.UVT = Var(
+                initialize=0.9,
+                bounds=(0, 1),
+                domain=NonNegativeReals,
+                units=pyunits.dimensionless,
+                doc="UV transmittance.",
+            )
+            self.A_coeff = Var(
+                initialize=2.15138466202152,
```

Comment:
With other models, it has been recommended to reduce the precision of these parameters to maybe 6 significant figures

Reply:
Good point, revised.